packages/app/components/Queue/TA/TAQueueListItem.tsx

```diff
@@ -19,6 +21,22 @@ const Container = styled.div<{ selected: boolean }>`
   align-items: flex-start;
 
   padding-top: 16px;
+  return string;
```

Comment:
Don't think this is meant to be here

Reply:
It's called in line 101

Reply:
Sorry I should have been more specific, I was talking about the `return string` part